File: Web Charge/Examples.Charge.Application/Dtos/PersonPhoneDto.cs
using System.ComponentModel.DataAnnotations;

namespace Examples.Charge.Application.Dtos
{
    public class PersonPhoneDto
    {
        public int BusinessEntityID { get; set; }
        public int PhoneNumberTypeID { get; set; }
        public string PhoneNumber { get; set; }
        public PersonDto Person { get; set; }
        public PhoneNumberTypeDto PhoneNumberType { get; set; }

    }
}

File: Web Charge/Example-Web/src/app/Dtos/person-phone-dto.ts
import { PersonDto } from "./person-dto";
import { PhoneNumberTypeDto } from "./phone-number-type-dto";

export class PersonPhoneDto {
    businessEntityID: number;
    phoneNumberTypeID: number;
    phoneNumber: string;
    person:PersonDto
    phoneNumberType:PhoneNumberTypeDto
}
  